src/telliot_feed_examples/cli/commands/report.py

```diff
@@ -102,13 +106,50 @@ def print_reporter_settings(
     click.echo(f"Gas price speed: {gas_price_speed}\n")
 
 
+def build_feed_from_input() -> Optional[DataFeed[Any]]:
```

Comment:
Can you move this function to a higher level in `/cli` so it can be used for a future command for just building query instances?